gson/pom.xml

```diff
@@ -7,6 +7,7 @@
     <version>2.8.1-SNAPSHOT</version>
   </parent>
 
+  <packaging>bundle</packaging>
```

Comment:
what is the impact of this change? By default the packaging is Jar. Is the output still Jar? If so, does the size change in anyway?